src/scripts/components/main/app/app.tsx

```diff
@@ -65,6 +71,27 @@ const TranslatedApp: FunctionComponent = () => {
         <Switch>
           <Route path="/" exact>
             {logo}
+            {showWelcomeScreen ? (
```

Comment:
I think we should abstract this more so that the onboarding and welcome stuff doesn't take up such a big part of the app component. Can you move this in a separate component which handles the state and css classes?

Reply:
And perhaps integrate the overlay directly into the components?

Reply:
Good point, will change it. 👍🏼